sqlglot/expressions.py

```diff
@@ -1862,22 +1863,54 @@ class CurrentDatetime(Func):
     arg_types = {"this": False}
 
 
+class CurrentTime(Func):
+    arg_types = {"this": False}
+
+
 class CurrentTimestamp(Func):
-    arg_types = {}
+    arg_types = {"this": False}
 
 
 class DateAdd(Func, TimeUnit):
     arg_types = {"this": True, "expression": True, "unit": False}
 
 
+class DateSub(Func, TimeUnit):
+    arg_types = {"this": True, "expression": True, "unit": False}
+
+
 class DateDiff(Func, TimeUnit):
     arg_types = {"this": True, "expression": True, "unit": False}
 
 
+class DateTrunc(Func, TimeUnit):
+    arg_types = {"this": True, "unit": True, "zone": False}
+
+
+class DateTimeAdd(Func, TimeUnit):
```

Comment:
should we change this to DatetimeAdd, notice the lower case t

Reply:
👍